pyproject.toml

```diff
@@ -29,7 +29,8 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.12"
-typer = "^0.12.5"
+tea-client = { git = "https://github.com/WrathfulSpatula/tea-client.git", branch = "develop" }
```

Comment:
Should we instead be pinning to the `tea-client` and `tea-console` repos we forked from UF? These are used elsewhere as dependencies already as well:
https://github.com/unitaryfund/tea-console
https://github.com/unitaryfund/tea-client

Reply:
Done.